packages/fastq_tools.py

```diff
@@ -0,0 +1,58 @@
+from typing import Union
+def count_gc(seq: str) -> float:
+    """
+    Counts gc content of sequence.
+    Arguments: 
+        -seq (str): dna/rna seqeunce.
+    Return: 
+        -float - percent of GC content in sequence.
+    """
+    gc = seq.lower().count('g') + seq.lower().count('c')
+    answer = gc / len(seq) * 100
+    return round(answer, 2)
+def count_qscore(score_string: str) -> float:
+    """
+    Counts quality score for inputed quality score string.
+    Argumets:
+        -score_string (str) - score string from fastq file.
+    Return:
+        -mean_quality (float) - mean quality of sequence based on quality score string.
+    """
+    total_score = 0
+    for symb in set(score_string):
+        total_score += score_string.count(symb) * (ord(symb) - 33)
+    return total_score / len(score_string)
+
+def fastq_tools(seqs: dict[str,str], gc_bounds: Union[int,tuple[int,int]] = (0,100), length_bounds: Union[int,tuple[int,int]] = (0, 2**32), quality_treshholds:int = 0) -> dict[str,str]:
```

Comment:
Согласно PEP8, лучше так: (обрати внимание на пробелы особенно, перенос по строкам тоже желателен)
```suggestion
def fastq_tools(
        seqs: Dict[str, str], 
        gc_bounds: Union[int, tuple[int, int]] = (0, 100), 
        length_bounds: Union[int, tuple[int, int]] = (0, 2**32), 
        quality_treshholds: int = 0
) -> Dict[str, str]:
```